x/concentrated-liquidity/position_test.go

```diff
@@ -642,7 +664,15 @@ func (s *KeeperTestSuite) TestDeletePosition() {
 				s.Require().Nil(positionIdBytes)
 			}
 
-			err = s.App.ConcentratedLiquidityKeeper.DeletePosition(s.Ctx, test.positionId, s.TestAccs[0], defaultPoolId)
+			if test.creator != nil {
+				defaultCreator = secondaryAccount
+			}
+
+			if test.poolId != defaultPoolId {
+				defaultPoolId = test.poolId
+			}
+
+			err = s.App.ConcentratedLiquidityKeeper.DeletePosition(s.Ctx, test.positionId, defaultCreator, defaultPoolId)
```

Comment:
What is the purpose of replacing defautls? We should be able to simply do:
```go
err = s.App.ConcentratedLiquidityKeeper.DeletePosition(s.Ctx, test.positionId, defaultCreator, defaultPoolId)
```

and se the desired `test.poolId` and `test.creator` for each test case

Reply:
yea that works too. changed